include/CLine3D.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "TVector3.h"
+
+class CLine3D
```

Comment:
I would improve readability by removing the 'C' prefix, as the coding convention proposed for sand-reco does not require it.

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules